SECURITY.md

```diff
@@ -4,13 +4,13 @@ Avalanche takes the security of the platform and of its users very seriously. We
 
 ## Reporting a Vulnerability
 
-**Please do not file a public ticket** mentioning the vulnerability. To disclose a vulnerability submit it through our [Bug Bounty Program](https://hackenproof.com/avalanche).
+**Please do not file a public ticket** mentioning the vulnerability. To disclose a vulnerability submit it through our [Bug Bounty Program](https://immunefi.com/bug-bounty/avalanche/information/).
```

Comment:
I think this is already updated in main. Could we pull those commits in so to reduce the diff in this PR?